apps/web/src/app/contact/page.tsx

```diff
@@ -1,24 +1,18 @@
 import Navbar from "@/components/shared/Navbar";
-import { Oswald } from "next/font/google";
 import c from "config";
 import { Instagram, Facebook, Twitter, Github } from "lucide-react";
 import Link from "next/link";
 
-const oswald = Oswald({
```

Comment:
non-blocking: Thanks for this but just for review purposes, would have been nice for this to be separate 👍